File: cpp/ApplicationManagers/Api/AottgApiError.c
// Type: ApplicationManagers.Api.AottgApiError
// Ghidra (System V/gcc) decompilation, IL2CPP structs + signatures applied.
// ---- AoTTG2 cross-reference ----
// Update status: unchanged since the previous dump
// C# structure: source/csharp/Scripts/ApplicationManagers.Api/AottgApiError.cs
// Prior real C# source: none
// --------------------------------

// ApplicationManagers.Api.AottgApiError$$Http
// il2cpp: System_String_o* ApplicationManagers_Api_AottgApiError__Http (int64_t statusCode, const MethodInfo* method);
// 0x44ce570

System_String_o * ApplicationManagers_Api_AottgApiError__Http(int64_t statusCode,MethodInfo *method)

{
  System_String_o *pSVar1;
  int64_t iStack_8;
  
  iStack_8 = statusCode;
  if (g_data_057aeb15 == '\0') {
    il2cpp_runtime_helper_023445d0(&"http_");
    g_data_057aeb15 = '\x01';
  }
  pSVar1 = System_Int64__ToString((int64_t)&iStack_8,(MethodInfo *)0x0);
  pSVar1 = System_String__Concat_3ae5ba0(_DAT_055de1b0,pSVar1,(MethodInfo *)0x0);
  return pSVar1;
}

File: csharp_body_deref/Scripts/ApplicationManagers.Api/AottgApiError.cs
// ==================== AoTTG2 cross-reference ====================
// Type: ApplicationManagers.Api.AottgApiError
// Update status: unchanged since the previous dump
// Real method bodies (native C decompile): source/cpp/ApplicationManagers/Api/AottgApiError.c
// Prior real C# source: none (type is newer than the last public source snapshot)
// Bodies below are empty IL2CPP metadata stubs; the logic lives in the native C.
// ===============================================================
using Il2CppDummyDll;

namespace ApplicationManagers.Api;

[Token(Token = "0x20007C6")]
public static class AottgApiError
{
	[Token(Token = "0x4002554")]
	public const string NetworkError = "network_error";

	[Token(Token = "0x4002555")]
	public const string BadResponse = "bad_response";

	[Token(Token = "0x4002556")]
	public const string Canceled = "canceled";

	[Token(Token = "0x6004A8D")]
	[Address(RVA = "0x43CE570", Offset = "0x43CE570", VA = "0x43CE570")]
	public static string Http(long statusCode)
	{
		return null;
	}
}
